### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by pier-oliviert** [pier-oliviert/phonebook#17]
There's some good documentation on the doc's site, but the README is kinda lacking. It'd be nice to better highlight the feature it supports as well as a quick way for people to try it out.

Right now, I feel like folks read the README and are asking themselves: "Now what?"

I'm opened to ideas on how to make the README better!

**Comment by ainsey11** [comment]
I agree, I've seen a few projects recently with some really slick readme files 

I might take this one on and see if we can make it a bit more friendly/snazzy and such, 

I'm thinking perhaps some more on a quick start, maybe a star graph etc, 

On a similar vein I'm wondering if we should do some issue templates for things like new provider requests, what do you think?
